cli/command/container/cp_test.go

```diff
@@ -181,7 +180,10 @@ func TestSplitCpArg(t *testing.T) {
 	for _, tc := range testcases {
 		tc := tc
 		t.Run(tc.doc, func(t *testing.T) {
-			skip.If(t, tc.os == "windows" && runtime.GOOS != "windows" || tc.os == "linux" && runtime.GOOS == "windows")
+			nonMatchingPlatform := tc.os == "windows" && runtime.GOOS != "windows" || tc.os == "unix" && runtime.GOOS == "windows"
```

Comment:
I was mostly concerned about splitting the OR'd parts into a separate variables to be honest 😅 

```go
    a := tc.os == "windows" && runtime.GOOS != "windows"
    b := tc.os == "unix" && runtime.GOOS == "windows"
    skip.If(t, a || b)
```

Mostly because it's easier to wrap your head about the end-result by looking at each of them separately, instead of trying to parse the whole 4 product expression.

Reply:
Split it over 2 checks (I tried to avoid the `skip.If(t, a || b)` because the output looks weird in the logs 😂

But again; happy to change if you prefer it differently ❤️